content/contributing/reference/triage.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "Triage"
+insert_anchor_links = "right"
+[extra]
+weight = 1
++++
+
+Bevy's scope and user base means that it gets a large volume of issues and pull requests. This work all needs to be organized: making it easier to search through to find related issues, to figure out where you can help, and to quickly communicate the status of a task at a glance. The continual process of organizing work is called **triage**.
+
+## Classifying issues and PRs
+
+Labels are our primary tool for organizing work. You can find a complete list with descriptions [here](https://github.com/bevyengine/bevy/labels). These are a few of the most common, organized by category:
+
+- **A**: Area (e.g. `A-Animation`, `A-ECS`, `A-Rendering`, ...).
+- **C**: Category. The most common include:
+    - `C-Bug`: unexpected or incorrect behavior.
+    - `C-Enhancement`: a new feature.
```

Comment:
not just a new feature, it can be an enhancement of an existing one

Reply:
```suggestion
    - `C-Enhancement`: a new feature or meaningful extension of an existing one.
```